### PR TITLE
Add characteristics to ability power roll

### LINKED ISSUE #92: Add characteristic(s) to ability power rolls
Need to define which characteristic(s) are used for the power roll. Some abilities allow a choice between multiple. Would need to determine the highest score to use when rolling and for damage.

### PR DESCRIPTION
**Pull request by cdverrett94**
Schema includes the characteristics that can be used for the ability. In _prepareCharacterData, the highest characteristic is chosen. Future work includes:

- Add checkbox to each tiers damage for adding characteristic to damage
- Add characteristic to damage if checked

Closes #92

### CODE REVIEW

**Comment by JPMeehan** [comment]
I think moving from using `choices` to providing the options as an argument to the form, as is done with `ClassModel` and `EquipmentModel`, is more conducive to people potentially modifying `characteristics`. Of course, it may be more conducive to just say "screw it" and move all that info into CONST and then hard lock choices.

File: src/module/data/item/ability.mjs [hunk 1]
```diff
@@ -60,6 +60,7 @@ export default class AbilityModel extends BaseItemModel {
 
     schema.powerRoll = new fields.SchemaField({
       enabled: new fields.BooleanField(),
+      characteristics: new fields.SetField(new fields.StringField({choices: ds.CONFIG.characteristics})),
```
Comment (JPMeehan) [inline src/module/data/item/ability.mjs:63]: Switch to `{required: true, blank: false}`

File: templates/item/partials/ability.hbs [hunk 2]
```diff
@@ -32,6 +32,7 @@
 </fieldset>
 {{formGroup systemFields.powerRoll.fields.enabled value=system.powerRoll.enabled}}
 {{#if system.powerRoll.enabled}}
+{{formGroup systemFields.powerRoll.fields.characteristics value=system.powerRoll.characteristics}}
```
Comment (JPMeehan) [inline templates/item/partials/ability.hbs:35]: Add `context.characteristics = Object.entries(ds.CONFIG.characteristics).map(([value], {label}) => ({value, label}));` to `getSheetContext` and then add `options=characteristics` in this

**Review by JPMeehan — CHANGES_REQUESTED** [review]

**Comment by cdverrett94** [comment]
I've made the changes you've requested. Good to know on options=characteristics and the rationale.

**Review by JPMeehan — APPROVED** [review]

**Pull request merged**